jscomp/main/melc.ml

```diff
@@ -454,7 +454,7 @@ let main: Melc_cli.t -> _ Cmdliner.Term.ret
     if version then print_version_string ();
     Ext_option.iter pp (fun pp -> Clflags.preprocessor := Some pp);
     if absname then Clflags.absname := absname;
-    Ext_option.iter bin_annot (fun bin_annot ->  Clflags.binary_annotations := not bin_annot);
+    Ext_option.iter bin_annot (fun bin_annot ->  Clflags.binary_annotations := bin_annot);
```

Comment:
I believe they're not actually on by default.  [This code](https://github.com/melange-re/melange/blob/cb7d065a5c1cf747c25063508035ff8a955a9bc1/jscomp/main/melc_cli.ml#L248-L257) should probably change to read:

```ocaml
let bin_annot =
  Arg.(
    value & vflag true
    & [
        (true,
          info ~doc:"Enable binary annotations (default: on)" [ "bin-annot" ] );
        (false,
          info ~doc:"Disable binary annotations (default: on)"
            [ "bs-no-bin-annot" ] );
      ])```

And we'd set `Clflags.binary_annotations` by default to `true` if the flag is missing. I currently [defaults to `false`](https://github.com/ocaml/ocaml/blob/4.14.0/utils/clflags.ml#L70)

Reply:
I believe default is true already, because of this line

https://github.com/melange-re/melange/blob/b1498afa30ff45c08a41f99d59621590719376ba/jscomp/core/bs_conditional_initial.ml#L56

Reply:
I suppose that also works